core/src/main/java/cucumber/runtime/arquillian/reporter/ReportAttributes.java

```diff
@@ -0,0 +1,40 @@
+package cucumber.runtime.arquillian.reporter;
+
+import java.util.Map;
+
+/**
```

Comment:
A detail but I try to keep the source code without @author etc (= fully given to the community and not make it belonging to developers). Any issue removing this comment?